src/KubernetesClient/KubernetesClient.csproj

```diff
@@ -20,7 +21,6 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.AspNetCore.JsonPatch" Version="1.1.2" />
-    <PackageReference Include="Nerdbank.GitVersioning" Version="2.2.33" PrivateAssets="all" />
```

Comment:
Why is this removed?

Reply:
change to azure pipeline versioning

```
  - task: DotNetCoreCLI@2
    displayName: 'dotnet pack'
    inputs:
      command: pack
      packagesToPack: src/KubernetesClient/KubernetesClient.csproj
      packDirectory: '$(Build.ArtifactStagingDirectory)/nupkg'
      majorVersion: 1
      minorVersion: 4      
      versioningScheme: byPrereleaseNumber
```

```
 this will generate a SemVer -compliant version formatted as X.Y.Z-ci-datetime where you choose X, Y, and Z.
```

The pro for generating version when CI is that it differentiate `build` and `commit` .
Same `commit` might be different due to `build` environment changed.